### DISCUSSION TITLE
GrapesJS LayoutEnhancer: Precision Spacing Tool

### DISCUSSION THREAD

**Discussion by GoodPHP** [GrapesJS/grapesjs#5809] [Show and tell]
![DALL·E-2024-03-23-13](https://github.com/GrapesJS/grapesjs/assets/22586552/d04364b9-3ab5-4efa-a9a9-c23e59fcaf03)

Demo:
https://devfuture.pro/live/margin-padding/

Buy:
https://gjs.market/products/grapesjs-layoutenhancer-precision-spacing-tool

The "GrapesJS LayoutEnhancer: Precision Spacing Tool" is a revolutionary plugin designed to empower web developers and designers with unprecedented control over the layout of their web pages. With a focus on enhancing the visual spacing between elements, this tool introduces a highly intuitive interface that allows for precise adjustments of margins and paddings, ensuring that every component of the web design is perfectly aligned and harmoniously spaced.

**Comment by mosh-tudor** [comment]
You have some bugs in the plugin that buyers need to be aware of.

- `!important` is evil
- undo manager doesn't work well (or at all?)
- this is not a valid css code:
  ```css
  .bdg-sect{
    padding:0px 166px !important 0 0px;
  }
  ```

Why don't you simply synchronize the padding/margin with the one in the editor? (style manager)
